.gitignore

```diff
@@ -1,6 +1,8 @@
 dist/
+types/
+lib/**/*.js
```

Comment:
@adambullmer Can you explain what this line is for?

Reply:
That's actually from some rapid iteration on getting this feature to work properly and no longer necessary. I'll remove to reduce confusion and unintended file ignoring